cc_types.c

```diff
@@ -746,6 +745,7 @@ struct type* create_enum(void)
 	}
 	else
 	{
+		maybe_bootstrap_error("non-anonymoous enum statement");
```

Comment:
There's an `o` too much in anonymous. I believe we should also add an `else` to the `if(match("=", global_token->s))` where we do `maybe_bootstrap_error` since that also isn't allowed.